Source/JavaScriptCore/shell/PlatformPlayStation.cmake

```diff
@@ -20,11 +20,15 @@ endif ()
 set(PLAYSTATION_jsc_PROCESS_NAME "JSCShell")
 set(PLAYSTATION_jsc_MAIN_THREAD_NAME "JSCShell")
 
-# Set the debugger working directory for Visual Studio
 if (${CMAKE_GENERATOR} MATCHES "Visual Studio")
+    # Set the debugger working directory for Visual Studio
     set_target_properties(jsc PROPERTIES VS_DEBUGGER_WORKING_DIRECTORY "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}")
-endif ()
 
+    # Set the startup target to JSC if WebCore disabled
+    if (NOT ENABLE_WEBCORE)
+        set_property(DIRECTORY ${PROJECT_SOURCE_DIR} PROPERTY VS_STARTUP_PROJECT jsc)
+    endif ()
```

Comment:
In this case jsc is only set as the startup project if `NOT ENABLE_WEBCORE`. It is not possible to build MiniBrowser without `ENABLE_WEBCORE` being `ON` so this means there will only ever be a single `VS_STARTUP_PROJECT` set.

Reply:
Ah, sorry. That's right. I miss understanding the case.